.mvn/wrapper/maven-wrapper.properties

```diff
@@ -1,6 +1,6 @@
 #
-# Copyright HomeAway, Inc 2018-Present. All Rights Reserved.
+# Copyright ExpediaGroup, Inc 2018-Present. All Rights Reserved.
```

Comment:
These need to be "Expedia, Inc" and should have an end year, as is specified in https://github.com/HotelsDotCom/hotels-oss-parent/blob/master/pom.xml#L282

Note the comma. I checked with legal last month and this is what they said to use.